app/controllers/api/base_controller/parameters.rb

```diff
@@ -82,6 +82,9 @@ def parse_filter(filter, operators)
                         [filter_value.gsub(/^'|'$/, ''), str_method]
                       when /^".*"$/
                         [filter_value.gsub(/^"|"$/, ''), str_method]
+                      when /^\d{4}-\d{2}-\d{2}/
```

Comment:
@abellotti MiqExpression does dates as YYYY-MM-DD and datetimes as YYYY-MM-DD HH:MM:SS though there's some nuance to it, i.e. zero-padding is optional, slashes can be used for dates, etc..